caption_cog.py

```diff
@@ -30,6 +30,7 @@
 from colorama import Fore, Style
 
 SUPPORTED_EXT = [".jpg", ".png", ".jpeg", ".bmp", ".jfif", ".webp"]
+Image.MAX_IMAGE_PIXELS = None
```

Comment:
Setting this to None may be legitimately dangerous.

For alignment to the main trainer, lets use `PIL.Image.MAX_IMAGE_PIXELS = 715827880*4`